src/main/java/org/dependencytrack/policy/cel/persistence/CelPolicyProjectRowMapper.java

```diff
@@ -51,7 +51,7 @@ public Project map(final ResultSet rs, final StatementContext ctx) throws SQLExc
         maybeSet(rs, "name", ResultSet::getString, builder::setName);
         maybeSet(rs, "version", ResultSet::getString, builder::setVersion);
         maybeSet(rs, "classifier", ResultSet::getString, builder::setClassifier);
-        maybeSet(rs, "is_active", ResultSet::getBoolean, builder::setIsActive);
+        maybeSet(rs, "inactive_since", RowMapperUtil::nullableTimestamp, builder::setInactiveSince);
```

Comment:
We can avoid a breaking change in the policy Protobuf schema by keeping `is_active`, and populating it based on whether `inactiveSince` is `null`. That would also solve the failing `buf` build check.